### PR TITLE
Autocomplete disabled

### PR DESCRIPTION
**Pull request by MaksymShokin**
@sofiiakvasnevska

Issue
Fliplet/fliplet-studio#5494

Description
Apparently this method works fine, although sometimes Chrome anyway autofills password. Maybe we should wait for Inna to test it before making PR in other components.

Screenshots/screencasts
autocomplete-off

Backward compatibility
This change is fully backward compatible.

### CODE REVIEW

File: templates/components/password.build.hbs [hunk 1]
```diff
@@ -2,7 +2,7 @@
   type="password"
   class="form-control"
   :readonly="autogenerate"
-  autocomplete="off"
+  autocomplete="new-password"
```
Comment (squallstar) [inline templates/components/password.build.hbs]: why `new-password`? If it needs to be a random field name then you can generate it with `Fliplet.guid()`

Comment (MaksymShokin) [reply templates/components/password.build.hbs]: This is the only way it works.

Comment (squallstar) [reply templates/components/password.build.hbs]: @maksim-an-upplabs can you elaborate? post relevant documentation to support the argument?

Comment (squallstar) [reply templates/components/password.build.hbs]: @MaksymShokin can you elaborate? post relevant documentation to support the argument?

Comment (MaksymShokin) [reply templates/components/password.build.hbs]: @squallstar 
After I tried other methods I found this article https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion.
Below it has section called "Preventing autofilling with autocomplete="new-password"."

**Pull request merged**